spec/serializer_spec.rb

```diff
@@ -193,17 +210,21 @@ def serialize_primary(object, options = {})
           },
           'relationships' => {
             'author' => {
-              'self' => '/posts/1/links/author',
-              'related' => '/posts/1/author',
-              'linkage' => nil,
+              'links' => {
+                'self' => '/posts/1/links/author',
+                'related' => '/posts/1/author'
+              },
+              'data' => nil,
             },
             'long-comments' => {
-              'self' => '/posts/1/links/long-comments',
-              'related' => '/posts/1/long-comments',
-              # Spec: Resource linkage MUST be represented as one of the following:
+              'links' => {
+                'self' => '/posts/1/links/long-comments',
+                'related' => '/posts/1/long-comments'
+              },
+              # Spec: Resource data MUST be represented as one of the following:
```

Comment:
Same here, this is a direct quote from the spec and it actually says "linkage" not "data". Please rename back to "Resource linkage MUST" here and elsewhere.